### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by ocisec**
Added Bastion draft page

### CODE REVIEW

File: cloud-infrastructure/infrastructure-security/bastion/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ 
+<Product description>
+ 
+# Table of Contents
+ 
+1. [Team Publications](#team-publications)
+2. [Useful Links](#useful-uinks)
+ 
+## Team Publications
+
```
Comment (AlexanderHodicke) [inline cloud-infrastructure/infrastructure-security/bastion/README.md:11]: Please pdate the link

**Review by AlexanderHodicke — CHANGES_REQUESTED** [review]
see my feedback

**Review by AlexanderHodicke — APPROVED** [review]

**Comment by AlexanderHodicke** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by ocisec** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**